base/musl/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  epics-base:
+    image: ${REGISTRY:-ghcr.io/cnpem}/lnls-alpine-3.18-epics-7:$TAG
+    build:
+      context: ../
+      dockerfile: musl/Dockerfile
+      labels:
+        org.opencontainers.image.revision: ${TAG}
+        org.opencontainers.image.source: ${SOURCE:-https://github.com/cnpem/epics-in-docker}
+        org.opencontainers.image.description: "EPICS base and modules build image"
```

Comment:
Maybe it deserves another description?

Reply:
I think so, yeah. Will try to come up with it.